packages/dappmanager/src/modules/compose/editor.ts

```diff
@@ -222,6 +222,13 @@ export class ComposeEditor {
     return parseUserSettings(this.compose);
   }
 
+  /**
+   * Returns compose network
+   */
+  getComposeNetwork(networkName: string): ComposeNetwork {
```

Comment:
You should return `: ComposeNetwork | null` and use null as indication that the network does not exist
```ts
return this.compose.networks?.[networkName] ?? null;
```